src/System.Runtime.Analyzers/UnitTests/AvoidZeroLengthArrayAllocationsTests.cs

```diff
@@ -34,7 +34,8 @@ public void EmptyArrayCSharp()
                 @"namespace System { public class Array { public static T[] Empty<T>() { return null; } } }";
 
             const string badSource = @"
-[System.Runtime.CompilerServices.Dynamic(new bool[0])] // no
```

Comment:
Why did we remove this? Its a compile error but we want to ensure analyzer doesn't complain for it.